tasks/hyrax-metadata-updates/tests/test-index.js

```diff
@@ -128,3 +129,58 @@ test('Test adding OPeNDAP URL to ECHO10 file with two OnlineResources', async (t
   const actual = addHyraxUrl(metadata, false, 'https://opendap.earthdata.nasa.gov/collections/C1453188197-GES_DISC/granules/GLDAS_CLSM025_D.2.0%3AGLDAS_CLSM025_D.A20141230.020.nc4');
   t.is(actual, expected.trim('\n'));
 });
+
+test('Test building CMR Search Params with short name and version', (t) => {
+  const inputSearchParams = {
+    shortName: 'GLDAS_CLSM025_D',
+    versionId: '2.0',
+  };
+  const expected = {
+    short_name: 'GLDAS_CLSM025_D',
+    version: '2.0',
+  };
+  const actual = getCmrSearchParams(inputSearchParams);
+  t.deepEqual(actual, expected);
+});
+
+test('Test building CMR Search Params with dataset ID', (t) => {
+  const inputSearchParams = {
+    datasetId: 'GLDAS_CLSM025_D.2.0',
+  };
+  const expected = {
+    dataset_id: 'GLDAS_CLSM025_D.2.0',
+  };
+  const actual = getCmrSearchParams(inputSearchParams);
+  t.deepEqual(actual, expected);
+});
+
+test('Test building invalid CMR Search Params with short name, version, and dataset ID', (t) => {
+  const inputSearchParams = {
+    datasetId: 'GLDAS_CLSM025_D.2.0',
+    shortName: 'GLDAS_CLSM025_D',
+    versionId: '2.0',
+  };
+
+  t.throws(
+    () => getCmrSearchParams(inputSearchParams),
+    {
+      message: 'Invalid list of keys for searchParams: dataset_id,short_name,version',
+      instanceOf: Error,
+    }
+  );
+});
+
+test('Test building invalid CMR Search Params with invalid params', (t) => {
```

Comment:
Should there be an additional unit test for the invalid params permutation datasetId and shortName?

Reply:
I think that's overkill, but I can ask him for one if you want.